src/core/models.py

```diff
@@ -700,7 +700,7 @@ def zookeeper_version(self) -> str:
     # retry to give ZK time to update its broker zNodes before failing
     @retry(
         wait=wait_fixed(5),
-        stop=stop_after_attempt(10),
+        stop=stop_after_attempt(3),
```

Comment:
VM one is 10. This affects UX more than it seems as it is related to deploy times. I would keep both the same, 3 or 10, whatever makes more sense.

Reply:
I'll make sure to switch the VM one also, as 50s was a long time I think.